package/Windows/Actions/stdafx.cpp

```diff
@@ -0,0 +1,4 @@
+#include "stdafx.h"
+
+// TODO: reference any additional headers you need in STDAFX.H
+// and not in this file
```

Comment:
What is the reason for this? Compilation speed? Is that really a TODO?

Reply:
> Compilation speed? 

Probably

> Is that really a TODO?

Probably not 😂 This is actually boilerplate generated by the project template. I'll just remove the todos (I don't think I even looked at this file before!).